workspace_host/interface.js

```diff
@@ -684,7 +696,7 @@ async function _autoUpdateJobManager() {
         debug(`watch: workspace_id=${workspace_id}, localPath=${local_path}`);
         const workspace = await _getWorkspaceAsync(workspace_id);
         const workspaceSettings = await _getWorkspaceSettingsAsync(workspace_id);
-        const s3_name = workspace.s3_name;
+        const remote_name = workspace.remote_name;
```

Comment:
In `_autoUpdateManager()`, don't we only want to take action for files for workspaces with S3 storage? I couldn't see where we were skipping files associated with FileSystem workspaces.

Reply:
I was thinking since the EFS/FileSystem jobs and the old S3 jobs will be running in different directories, the file watcher won't be tracking the FileSystem jobs.  So these files shouldn't ever show up in the update queue/update manager.